src/ledger/LedgerManagerImpl.cpp

```diff
@@ -2875,15 +2760,8 @@ LedgerManagerImpl::sealLedgerTxnAndStoreInBucketsAndDB(
     // protocol version prior to the upgrade. Due to this, we must check the
     // initial protocol version of ledger instead of the ledger version of
     // the current ltx header, which may have been modified via an upgrade.
-    sealLedgerTxnAndTransferEntriesToBucketList(
-        ltx, ledgerCloseMeta, ltx.loadHeader().current(), initialLedgerVers);
-    if (ledgerCloseMeta &&
-        protocolVersionStartsFrom(initialLedgerVers, SOROBAN_PROTOCOL_VERSION))
-    {
-        ledgerCloseMeta->setNetworkConfiguration(
-            mApplyState.getSorobanNetworkConfigForCommit(),
-            mApp.getConfig().EMIT_LEDGER_CLOSE_META_EXT_V1);
-    }
+    sealLedgerTxnAndTransferEntriesToBucketList(ltx, ledgerCloseMeta,
```

Comment:
Could we please rename sealLedgerTxnAndTransferEntriesToBucketList to "transferToBucketListAndPopulateInMemoryState"? I had a hard time figuring out what's going on here, because we have two functions named `sealLedgerTxnAndStoreInBucketsAndDB` and `sealLedgerTxnAndTransferEntriesToBucketList`, where names suggest they do the same thing (they don't)